<?xml version="1.0" encoding="utf-8"?>

<launch>

  <!-- Load Parameter -->
  
  <param name="yolov5_path"       value="/home/bei/robot_ws/src/yolov5_ros/yolov5"/>
  <param name="use_cpu"           value="false" />

  <!-- Start yolov5 and ros wrapper -->
  <node pkg="yolov5_ros" type="yolo_v5.py" name="yolov5_ros" output="screen" >
    <param name="weight_path"       value="/home/bei/robot_ws/src/yolov5_ros/weights/yolov5s.pt"/>
    <!-- <param name="image_topic"       value="/usb_cam/image_raw" /> -->
    <param name="image_topic"       value="/kinect2/hd/image_color" />
    <param name="pub_topic"         value="/yolov5/BoundingBoxes" />
    <param name="camera_frame"       value="camera_color_frame"/>
    <param name="conf"              value="0.3" />
  </node>

</launch>
